src/components/map/map.tsx

```diff
@@ -169,6 +223,50 @@ export class Map implements GxComponent {
     }
   }
 
+  private updateSelectionMarkerPosition() {
+    const centerCoords = this.map.getCenter();
+    this.centerCoords = `${centerCoords.lat},${centerCoords.lng}`;
+    this.selectionMarker.setAttribute("coords", this.centerCoords);
+  }
+
+  private selectionLayerEvents() {
```

Comment:
Maybe we can give this method a better name, that reflects the action taken with the selection layer events.

Reply:
Yas. it needs more clues about its purpose, but... what name? `selectionLayerMapEventsSubscriber`?

Let's see: That method evaluates the state of selectionLayer (if it is `true` or `false`) and subscribes or unsubscribes the map to events emitted by leaflet and give certain behavior depending in the event listened
So, what do you think to call it `selectLay_EventManager`?

Reply:
I usually start methods with a verb. In this case, I'd use: `registerSelectionLayerEvents`

Reply:
defining name of methods with a verb 🤔 That was a good one. I will remember that.

Reply:
renamed!
Look in comit be8cd25